gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenAuthorityService.java

```diff
@@ -311,4 +327,8 @@ else if (! (key instanceof RSAPrivateKey)) {
   @Override
   public void stop() throws ServiceLifecycleException {
   }
+
+  protected Optional<String> getCachedSigningKeyKID() {
```

Comment:
"KID" is an acronym for "KeyID", correct? If so, then I think this should be named `getCachedSigningKeyID()`

Reply:
Yup, I'll change it!